samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/services/assessment/PublishedAssessmentService.java

```diff
@@ -362,7 +362,17 @@ public Integer getTotalSubmission(String agentId, String publishedAssessmentId)
         getPublishedAssessmentFacadeQueries().
         getTotalSubmission(agentId, new Long(publishedAssessmentId));
   }
-
+  
+  /**
+   * return submission numbers for the assessment by giving the publishedAssessmentId
+   * Added by -Qu for assessment deletion safe check SAM-2587
+   */
```

Comment:
Can you add @param and @returns for this JavaDocs like you did for the others? And you can remove your name here too.